src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.csproj

```diff
@@ -8,15 +11,18 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Dynamo.WorkspaceDependency</RootNamespace>
     <AssemblyName>WorkspaceDependencyViewExtension</AssemblyName>
+    <EnableDefaultPageItems>false</EnableDefaultPageItems>
```

Comment:
did this give you errors in the net6 builds if not added?

Reply:
Yes.